pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -47,9 +47,7 @@ type Autoscaler struct {
 	lister       corev1listers.EndpointsLister
 	reporterCtx  context.Context
 
-	// State in panic mode. Carries over multiple Scale calls. Guarded
-	// by the stateMux.
-	stateMux     sync.Mutex
```

Comment:
Let's document in the Scale function's comment then that it is not threadsafe and to be called from just one goroutine.

Reply:
Done.